tools/interactive/interactivetool_askomics.xml

```diff
@@ -1,26 +1,44 @@
-<tool id="interactive_tool_askomics" tool_type="interactive" name="AskOmics" version="0.1">
+<tool id="interactive_tool_askomics" tool_type="interactive" name="AskOmics" version="1.0">
     <description>a visual SPARQL query builder</description>
     <requirements>
-        <container type="docker">quay.io/askomics/askomics-ie:17.12_g19.09</container>
+        <container type="docker">askomics/flaskomics-with-dependencies:3.1.1</container>
```

Comment:
While we're at it, we should update to 3.2.0

Reply:
Does the image exist? i.e. is it only change on this line?

Reply:
yes, [askomics/flaskomics-with-dependencies:3.2.0 exists](https://hub.docker.com/layers/askomics/flaskomics-with-dependencies/3.2.0/images/sha256-c5f70e8d2ed29a96c8a825299c60063d68254d650a4bc21ea82f0bf1bc68f388), we already use it on galaxy.genouest.org